docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.traditional-pairings.scss

```diff
@@ -67,6 +70,7 @@ $ht-traditional-pairings: (
     'tertiary-reversed': #f4f4f4,
     'tertiary-darken-20': darken(#8c1515, 20%),
     'spotlight': #e0e0d1,
+    'light-well': #ecece4,
   ),
   'vireo': (
```

Comment:
I know it wasn't in the designs, but we need to define colors for the new `vireo` pairing. Please check with Katy to choose the color.